.gitignore

```diff
@@ -15,6 +15,7 @@
 clifton.sh
 dump.rdb
 yarn-error.log
+.DS_Store
```

Comment:
I'm generally of the opinion that system-specific `.gitignore` content should be in the developer's global `.gitignore` file rather than the project file (that's where I ignore `.DS_Store`). But if the team is mostly on macOS, it's not hurting to have it here I suppose.